Jasper Overall/JS Homework 01/homework.js

```diff
@@ -0,0 +1,60 @@
+console.log('Is this thing on?');
+//CACULATOR PART 1
+
+//Solution
+
+function squareNumber(num) {
```

Comment:
This function declaration works, but we recommend writing it like: 
const squareNumber = function (num) {